lib/enkf/block_obs.cpp

```diff
@@ -358,11 +348,11 @@ double block_obs_chi2(const block_obs_type * block_obs, const void * state, node
 /**
    The index is into the the number of active cells which are observed by this observation.
 */
-void block_obs_iget(const block_obs_type * block_obs, int index , double *value , double * std) {
-  const point_obs_type * point_obs = block_obs_iget_point_const( block_obs , index );
-  *value = point_obs->value;
-  *std   = point_obs->std;
-}
+//void block_obs_iget(const block_obs_type * block_obs, int index , double *value , double * std) {
```

Comment:
I realise this is a draft, but just pointing out the commented-out code here just in case.